usertable.cpp

```diff
@@ -432,10 +432,9 @@ void UserTable::OnEvent(InotifyEvent& rEvt)
           oldpos = pos + 2;
         }
         else if (cs[px] == '&') {     // numeric mask
-          char* s;
-          asprintf(&s, "%u", (unsigned) rEvt.GetMask());
+          char s[11];
```

Comment:
Mmmm... 11? How come? I'll need to see what `rEvt.GetMask()` can return.

Why did you prefer `char s[11]`? May be better to use `char* s = new char[SOME_SIZE]`, then verify that `s != NULL`.